internal/email/email.go

```diff
@@ -61,56 +99,88 @@ func NewMessage(
 	return message.NewMessage(id.String(), payload), nil
 }
 
-// getBodyHTML returns the HTML body for the email based on the message payload
-func getEmailBodyHTML(ctx context.Context, inviteURL, minderURL, sponsor, project, role, inviteeEmail string) string {
-	data := struct {
-		AdminName        string
-		OrganizationName string
-		InvitationURL    string
-		RecipientEmail   string
-		MinderURL        string
-		TermsURL         string
-		PrivacyURL       string
-		SignInURL        string
-		RoleName         string
-		RoleVerb         string
-	}{
-		AdminName:        sponsor,
-		OrganizationName: project,
-		InvitationURL:    inviteURL,
-		RecipientEmail:   inviteeEmail,
-		MinderURL:        minderURL,
-		TermsURL:         DefaultMinderTermsURL,
-		PrivacyURL:       DefaultMinderPrivacyURL,
-		SignInURL:        minderURL,
-		RoleName:         role,
-		RoleVerb:         authz.AllRolesVerbs[authz.Role(role)],
+// getHTMLBodyString returns the HTML body for the email based on the message payload.
+// If there is an error creating the HTML body, it will try to return the text body instead
+func getEmailBodyHTML(ctx context.Context, data bodyData) string {
```

Comment:
Why does this function not return an error?  In particular, it seems like we want to distinguish between "template rendered successfully" and "template failed rendering" before we send the email.

Reply:
Fixed it 👍